styleguide/source/assets/scss/02-molecules/_hero-evergreen.scss

```diff
@@ -0,0 +1,93 @@
+.hero-evergreen {
+  display: flex;
+  flex-flow: column wrap;
+
+  @include breakpoint($bp-med) {
+    flex-flow: row nowrap;
+  }
+
+  &__mobile-image {
+    display: block;
+
+    @include breakpoint($bp-small) {
+      display: none;
+    }
+  }
+
+  &__aside {
+    @include gutter-all(padding, $gutter / 2, $gutter-mobile / 2);
+    background: $purple;
+    flex-basis: 33.33%;
+    color: $white;
+
+    &--full-width {
+      flex-basis: 100%;
+    }
+  }
+
+  &__background {
+    flex-basis: 66.66%;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    background-size: cover;
+
+    @include breakpoint($bp-small max-width) {
+      background: none !important;
+    }
+
+    .hero-evergreen__buttons {
+      display: flex;
+      background: $white;
+      padding-top: $gutter;
```

Comment:
There are 2 padding-tops here

padding-top: $gutter;
padding-top: 15px;

Reply:
This is corrected.